### PR TITLE
insert current path in syspath whenever cli invoked [WIP]

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- always include current path in syspath for CLI so that custom channels, policies, and components can be found
- fix https://github.com/RasaHQ/rasa/issues/3883

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by tmbo** [comment]
@erohmensing please assign the issues to yourself and move them to doing before you work in them :wink: (I was just about to assign someone else until I saw your PR)

**Comment by erohmensing** [comment]
@tmbo I assigned it to myself. I left it in discussion bc we hadn't discussed it yet, though

**Comment by akelad** [comment]
@erohmensing you're supposed to assign issues to yourself that are in the backlog, not in the discussion column 😂i'm not gonna review this yet because tobi and i weren't sure whether we actually want to do that

**Comment by erohmensing** [comment]
my b 😅just was working on a similar issue, saw it and went for it. as i mentioned in the issue, assigned before i noticed the discusison tag

File: rasa/cli/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import os
+import sys
+
+# insert current path in syspath so module is found
+sys.path.insert(1, os.getcwd())
```
Comment (wochinge) [inline rasa/cli/__init__.py]: I think it would be safer if we put it in `main()` of the module `__main__`. What do you think?

Comment (erohmensing) [reply rasa/cli/__init__.py]: yeah that seems to make more sense -- will be easier for others to notice if they're debugging something funky that happens 👍

**Pull request merged**